speech_assets/intentSchema.json

```diff
@@ -33,6 +33,12 @@
             ]
         },
         {
+            "intent": "GeeMusicPlayPlaylistIntent",
+            "slots": [
+                {"name":"playlist_name", "type":"AMAZON.MusicRecording"}
```

Comment:
This should be [AMAZON.MusicPlaylist](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/built-in-intent-ref/slot-type-reference#musicplaylist)